### DISCUSSION TITLE
Craft 4 upgrade - issue with viewing Freeform submission

### DISCUSSION THREAD

**Discussion by kate-stretton** [solspace/craft-freeform#902] [Questions & Answers]
**Describe the bug or issue you're experiencing**
I'm experience an issue with Freeform that I'm hoping you can please help with. 
I've upgraded a site from Craft 3.9.4 to Craft 4.5.6.1. As part of this, I've also updated Freeform from 3.13.32 to 4.1.9. 
When I try to view a form submission, only some of the fields are visible (the other fields briefly flicker on the screen and then disappear).  The information I submitted in the form is being saved in the database correctly. 

**Steps to reproduce**

1. Under Freeform, click 'Submissions' 
2. Open a submission

**Expected behavior**
I expected to see all form fields with the information I saved on submission.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.5.6.1.
- Freeform Version: 4.1.9.
- Freeform Edition: Pro- Fresh Install or Upgrade: Upgrade

**Additional context**
Here's a screenshot of the system report for this site: 

![image](https://github.com/solspace/craft-freeform/assets/81283807/692005a2-5a88-44da-8218-4c3a54575a9c)

**Comment by kjmartens** [comment]
Sorry for the trouble you're experiencing @kate-stretton.

I have a couple of questions that I hope will help clarify your setup:

1. Is there any chance you have conditional rules configured for the form? The submission detail page in the CP will show/hide fields based on conditional rules for the given submission data (which would appear as a quick flicker of the field)
2. Is this happening for all forms, or just some of them?

**Comment by kate-stretton** [comment]
Hi @kjmartens, thanks for taking a look at this. 

Yes, I do have conditional rules configured for the forms. Some of the fields that flicker briefly should be hidden based on the conditional rules, so that makes sense. However, the rest of the fields should be displayed based on the conditional rules. 

Yes, this is happening for all forms.

Thanks!

**Comment by kjmartens** [comment]
Thanks for the additional information, @kate-stretton.

Can you share with me either a video or a series of screenshots that show:

1. The form inside the form builder.
2. The conditional rules set for it.
3. A CP submission detail page that is affected?

I haven't heard anyone report this issue before, and I can't duplicate it myself, so perhaps there's something that will become more obvious once I see that info. 🙂
